windows-gaming-driver/src/config.rs

```diff
@@ -252,6 +250,13 @@ pub struct StorageDevice {
 }
 
 #[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct VfioDevice {
+    pub resettable: bool,
+    pub slot: String,
+    pub device: PciId,
```

Comment:
This is what it would currently look like in the config:
(Assuming github would respect the indentation)

>   pci_devices: 
>     - 
>       resettable: false
>       slot: "0000:02:00.0"
>       device: 
>         vendor: 4318
>         device: 6918

Having 

>   pci_devices: 
>     - 
>       resettable: false
>       slot: "0000:02:00.0"
>       id: 
>         vendor: 4318
>         device: 6918

feels strange to me as well